arbos/arbosState/arbosstate.go

```diff
@@ -308,6 +311,12 @@ func InitializeArbosState(stateDB vm.StateDB, burner burn.Burner, chainConfig *p
 		return nil, err
 	}
 
+	transactionCensorsStorage := sto.OpenCachedSubStorage(transactionCensorSubspace)
+	err = addressSet.Initialize(transactionCensorsStorage)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Do we also need to do this when upgrading arbos to v60 on existing chains? Maybe not, but I want to be sure.